configure.ac

```diff
@@ -0,0 +1,66 @@
+m4_define([mender_version], m4_normalize(m4_esyscmd([git describe --exact HEAD 2>/dev/null || git rev-parse --short HEAD])))
+AC_INIT([mender], [mender_version], [contact@mender.io])
```

Comment:
Oh, forgot. Should we make it:
```
AC_INIT([mender-client],...)
```
Or something other than just mender?

@kacf @vpodzime

Reply:
`mender` for now, we will see how this is handled in CMake and where/how far it actually propagates.